modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -440,11 +457,47 @@ public void testSendNotificationWithAccountRoles() throws Exception {
 
 		// Send email with an object definition not restricted by account entry
 
```

Comment:
@devpedrotavares please move this comment to be right before calling the `_testSendNotificationWithRoles`, i.e., after creating all users